go.sum

```diff
@@ -73,6 +73,7 @@ github.com/golang/protobuf v1.3.1/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5y
 github.com/golang/protobuf v1.3.2 h1:6nsPYzhq5kReh6QImI3k5qWzO4PEbvbIW2cwSfR/6xs=
```

Comment:
Can you run `go mod tidy` and see if this removes these changes?

Reply:
It did not--it created a different diff.